### DISCUSSION TITLE
Poll mock library

### DISCUSSION THREAD

**Discussion by thebino** [AndroidDev-social/DodoForMastodon#55] [Polls]
As discussed in #46 we need to find a mocking solution for tests in this project.
For details check https://github.com/AndroidDev-social/MastodonCompose/discussions/46 before giving your vote.

**Comment by bitPogo** [comment]
Just a note - Mockactive & KMock target Multiplatform as well.

**Reply by thebino** [reply]
thanks for pointing this out, I missed adding this to the answers. But editing the vote will reset the votes.
